src/interchange/src/avro.rs

```diff
@@ -163,7 +163,8 @@ mod tests {
         ];
         for (typ, datum, expected) in valid_pairings {
             let desc = RelationDesc::empty().with_column("column1", typ.nullable(false));
-            let schema_generator = AvroSchemaGenerator::new(None, None, None, desc, false).unwrap();
+            let schema_generator =
+                AvroSchemaGenerator::new(None, None, None, desc, false, false).unwrap();
```

Comment:
This is not your original sin, but this function could use some love––a config struct or something could make it a little less opauqe.

Reply:
done